text2chem/regex_parser.py

```diff
@@ -75,13 +75,13 @@ def make_fraction_convertion(formula):
         :param formula:
         :return:
         """
-        re_a = r"([0-9\.]*)"
-        re_b = r"(\([0-9\.]*)"
+        re_a = r"([0-9.]*)"
```

Comment:
Could you please explain the change in this file? Just to be sure I understood correctly